src/test/java/lotto/LottoGameTest.java

```diff
@@ -39,12 +25,11 @@ void gameStartAndcalculateReturnRate() {
         tickets.add(new LottoTicket(Arrays.asList(11, 12, 13, 14, 15, 16)));
 
         LottoGame lottoGame = new LottoGame(new LottoTickets(tickets));
-        lottoGame.gameStart(winningNumbers);
+        GameResult gameResult = lottoGame.gameStart(winningNumbers);
 
-        double rate = lottoGame.calculateReturnRate();
-        assertThat(lottoGame.getResults().get(Rank.FOURTH)).isEqualTo(1);
-        assertThat(lottoGame.getResults().get(Rank.MISS)).isEqualTo(1);
-        assertThat(lottoGame.calculateReturnRate()).isEqualTo(2.5);
+        assertThat(gameResult.getResultStats().get(Rank.FOURTH)).isEqualTo(1);
+        assertThat(gameResult.getResultStats().get(Rank.MISS)).isEqualTo(1);
+        assertThat(gameResult.getReturnRate()).isEqualTo(2.5);
```

Comment:
객체간 비교를 해보면 어떨까요?
```suggestion
        GameResult expected = new GameResult(...);
        assertThat(gameResult).isEqualTo(expected);
```
중요한 부분은 아니지만 연습삼아 도전해 보시면 좋을 것 같아요.